standalone/pruntime/enclave/src/contracts/mod.rs

```diff
@@ -46,6 +46,8 @@ impl AccountIdWrapper {
             .context("Failed to decode AccountId hex")?;
         Self::try_from(&bytes)
     }
+
+    #[allow(clippy::inherent_to_string)]
```

Comment:
AccountIdWrapper is not needed I think?

Reply:
> AccountIdWrapper is not needed I think?

I thought exactly the same, there is a [TODO](https://github.com/Phala-Network/phala-blockchain/pull/392/files/e13b5377fb323fd29b210a202eeefb5243682774#diff-7f0dd15b7961dfc20128084d8246f4bf96eabe9902a8119ae6c8c36f35248d62R31)